CHANGES/669.misc

```diff
@@ -1 +1 @@
-move update type detecting to Update
+Update type detection is was moved from Dispatcher to Update object
```

Comment:
Fix typo or mistake, whatever it is.

```suggestion
Update type detection was moved from Dispatcher to Update object
```

I'd also prefer putting the verb first, like in (imo good) commit messages.

```suggestion
Moved update type detection from Dispatcher to Update object
```